modules/ossm-multi-cluster-configuration-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// This module is used in the following assemblies:
+
+// * install/ossm-multi-cluster-topology.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-multi-cluster-configuration-overview_{context}"]
+= Multi-Cluster configuration overview 
+
+To configure a multi-cluster topology you must:
```

Comment:
ISG: Avoid continuing a lead-in sentence into its list items:
```suggestion
To configure a multi-cluster topology you must perform the following actions:
```
or similar.

Reply:
done